### DISCUSSION TITLE
Επιλογή ώρας για την επόμενη ΟΣΣ (Σάββατο 20/5/23)

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60/syllabus#75] [Polls]
Θα γίνει σύντομη συμπληρωματική ΟΣΣ το Σάββατο 20/5 με σκοπό το σχολιασμό της 5ης ΓΕ με έμφαση στην προσομοίωση της εξέτασης και θα δοθούν διευκρινίσεις και οδηγίες από τον καθηγητή.

**Comment by afoscolou** [comment]
Κύριε Χωριανόπουλε,

Η πλειοψηφία όσων συμμετείχαν στην ψηφοφορία για την ώρα της επόμενης ΟΣΣ στο μάθημά σας επέλεξε τις 16:00 το Σάββατο 20/5, εάν κι εσείς μπορείτε.

Ευχαριστώ

**Reply by epidrome** [reply]
OK!
